digitalocean/resource_digitalocean_kubernetes_cluster_test.go

```diff
@@ -276,6 +278,7 @@ provider "kubernetes" {
   cluster_ca_certificate = base64decode(
     digitalocean_kubernetes_cluster.foobar.kube_config[0].cluster_ca_certificate
   )
+  token = digitalocean_kubernetes_cluster.foobar.kube_config[0].token
```

Comment:
If tokens are the way to do this going forward, can you also update the example in the documentation?

https://github.com/terraform-providers/terraform-provider-digitalocean/blob/master/website/docs/r/kubernetes_cluster.html.markdown#example-usage

Reply:
✅ https://github.com/terraform-providers/terraform-provider-digitalocean/pull/309/commits/6b4908ba0dbb6a6ebd4e18fdd62cdd439ff7a372

Reply:
✅ https://github.com/terraform-providers/terraform-provider-digitalocean/pull/309/commits/6b4908ba0dbb6a6ebd4e18fdd62cdd439ff7a372